prow/config.yaml

```diff
@@ -4677,6 +4678,7 @@ periodics:
         hostPath:
           path: /mnt/disks/ssd0
   - name: ci-kubernetes-e2e-kubeadm-gce-ipvs
+    max_concurrency: 1
```

Comment:
discussed offline but noting here, I don't think we want max_concurrency for this, it will just build up a backlog of waiting prowjobs, we just want to fix the job interval to something we can handle without needing this.